communications/meetings.md

```diff
@@ -21,7 +21,7 @@ Guide for Organizers:
 Guide for Participants:
 - Go through relevant context, examples (where applicable):
     - Meeting notes and their associated pre-meeting discussions prepared by the organizer
-    - Summary of pre-meeting discussions through the use of the `/notes` command. ![notes-discussion-summary](https://user-images.githubusercontent.com/2837532/103942564-9bc18b80-50fe-11eb-9c3b-12dde574c43e.png)
+    - Summary of pre-meeting discussions through the use of the [`/notes`](../workspace-automation.md#notes) command. ![notes-discussion-summary](https://user-images.githubusercontent.com/2837532/103942564-9bc18b80-50fe-11eb-9c3b-12dde574c43e.png)
```

Comment:
lol the link goes to `/demo` for some random reason 🤷

Reply:
i see `/notes` clicked from https://github.com/EQWorks/common/blob/18e727afe99306877a112918565fbac057420666/communications/meetings.md